### ISSUE TITLE
Undo (and redo) in viewer

### ISSUE THREAD

**Issue by roym899** [rerun-io/rerun#3135]
**Is your feature request related to a problem? Please describe.**
Sometimes I accidentally remove a space view, making it quite annoying to set it up again.
Similarly, sometimes I move a view around which destroys the tiling, but there's no way to undo without setting everything up from scratch.

**Describe the solution you'd like**
Undoing changes in the viewer (like removing and moving a view) with Ctrl+Z.

Redo would be nice to have as well, but not as important.

Related to #1132.

**Comment by emilk** [comment]
* Blocked by https://github.com/rerun-io/rerun/issues/8056
